### DISCUSSION TITLE
KOreader cannot adust color temperature on Kobo Libra 2

### DISCUSSION THREAD

**Discussion by stevenzhgl** [koreader/koreader#8679] [Q&A]
Koreader cannot  adjust automatically color temperature for blue light reduction on Kobo Libra 2

**Comment by NiLuJe** [comment]
There's a plugin for that (I, err, don't remember on which name we settled, though :D), make sure it's enabled and set up properly?

**Comment by stevenzhgl** [comment]
Thanks!&nbsp; I got it!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "koreader/koreader"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月18日(星期二) 凌晨5:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [koreader/koreader] KOreader cannot adust color temperature on Kobo Libra 2 (Discussion #8679)





 
There's a plugin for that (I, err, don't remember on which name we settled, though :D), make sure it's enabled and set up properly?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
